model/src/main/java/com/basistech/rosette/dm/internal/TextWrapper.java

```diff
@@ -40,6 +40,7 @@
  * on token indexes.  The underlying <code>Text</code> object contains only
  * character offsets.
  */
+@SuppressWarnings("deprecation")
```

Comment:
It's actually not quite like that. The test in the internal repo is really a test of the ara stuff, it can't just be moved here. There is, in fact, a test of the same name here in this repo.